golangcilint.yaml

```diff
@@ -0,0 +1,75 @@
+run:
+  timeout: 60m
+
+linters:
+  enable-all: true
```

Comment:
How do you feel about not tweaking these?  While I don't some of the defaults, it might be easier than keeping a list of what we don't want?